### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by spithash**
Updated Copyright information

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -631,8 +631,8 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    {one line to give the program's name and a brief idea of what it does.}
-    Copyright (C) {year}  {name of author}
+    Netdata
+    Copyright (C) 2020  Netdata, Inc.
```
Comment (ilyam8) [inline LICENSE:635]: it is `Netdata, Inc.` since 2018 (i checked nd dashboard)

**Comment by ilyam8** [comment]
thx @spithash 👋

File: LICENSE (hunk 1 above)
Comment (spithash) [reply LICENSE]: ah cool, thanks :+1:

**Comment by OdysLam** [comment]
Thanks @spithash and welcome to our little community!

If you have any question, just shoot it away in our forums at https://community.netdata.cloud!

**Comment by ilyam8** [comment]
@cakrit please take a look

**Comment by Ferroin** [comment]
I’m curious why we think we need to update the generic disclaimer templates in our license file?

These do not indicate any copyright at all, they’re just sample text that is intended to be templated out accordingly _when it gets used_.

**Comment by ilyam8** [comment]
> These do not indicate any copyright at all, they’re just sample text that is intended to be templated out accordingly when it gets used.

@cakrit please take a look, perhaps it is `wontfix` and we can close the PR

**Comment by Ferroin** [comment]
> > These do not indicate any copyright at all, they’re just sample text that is intended to be templated out accordingly when it gets used.
> 
> @cakrit please take a look, perhaps it is `wontfix` and we can close the PR

For the record, if we _do_ update this, it should be something along the lines of ‘2018 onwards’, we have made code changes since 2018, ergo the copyright is not _just_ in 2018.

**Comment by ilyam8** [comment]
closing based on https://github.com/netdata/netdata/pull/10051#issuecomment-772487609

**Pull request closed without merging**